File: Task 2/Java/app/src/main/java/dev/ogabek/java/model/Note.java
package dev.ogabek.java.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Note extends RealmObject {

    @PrimaryKey
    private int id;
    private String text;

    public Note(){

    }

    public Note(int id, String text) {
        this.id = id;
        this.text = text;
    }

    public int getId() {
        return id;
    }

    public String getText() {
        return text;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: Task 2/Kotlin/app/src/main/java/dev/ogabek/kotlin/model/Note.kt
package dev.ogabek.kotlin.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Note : RealmObject {

    @PrimaryKey
    var id: Int = 0
    var text: String? = null

    constructor()

    constructor(id: Int, text: String) {
        this.id = id
        this.text = text
    }

}